static/app/views/dashboards/widgetCard/issueWidgetCard.spec.tsx

```diff
@@ -94,20 +110,23 @@ describe('Dashboards > IssueWidgetCard', function () {
   it('renders with title and issues chart', async function () {
     MemberListStore.loadInitialData([user]);
     render(
-      <WidgetCard
-        api={api}
-        organization={organization}
-        widget={widget}
-        selection={selection}
-        isEditingDashboard={false}
-        onDelete={() => undefined}
-        onEdit={() => undefined}
-        onDuplicate={() => undefined}
-        renderErrorMessage={() => undefined}
-        showContextMenu
-        widgetLimitReached={false}
-        widgetLegendState={widgetLegendState}
-      />
+      <DashboardsMEPProvider>
```

Comment:
Do you need to wrap it here if you're already passing it down with `wrapper: Wrapper`?

Reply:
Good catch of bad copypasta! Removed.